cmd/flowserver/static/start.json

```diff
@@ -37,7 +37,7 @@
             "timezone": "America/Guayaquil",
             "urns": [
                 "tel:+12065551212",
-                "facebook:macklemore"
```

Comment:
nothing against macklemore - this just isn't a valid facebook URN

Reply:
:(